tests/testthat/test-store.R

```
@@ -131,3 +131,18 @@ test_that("additional columns", {
   val <- dbGetQuery(store@.con, "select text, h1 from chunks")
   expect_equal(nrow(val), 2)
 })
+
+test_that("Allow a NULL embedding function", {
+  store <- ragnar_store_create(embed = NULL)
+  chunks <- data.frame(
+    origin = c("foo", "bar"),
+    hash = c("foo", "bar"),
+    text = c("foo", "bar")
+  )
+  ragnar_store_update(store, chunks)
+  expect_error(ragnar_store_build_index(store)) # must remove vss from embeddings
```

Should we throw an error here? Maybe it would be better to let `ragnar_store_build_index()` gracefully accommodate this scenario; the requirement to use `ragnar_retrieve_bm25()` instead of `ragnar_retrieve()` is probably enough of a guardrail.